scylla/assets/monitors/instance_down.json

```diff
@@ -0,0 +1,25 @@
+{
+	"name": "[Scylla] Server is shutting down",
+	"type": "metric alert",
+	"query": "avg(last_1m):max:scylla.node.operation_mode{*} > 3",
+	"message": "{{server.name}} is shutting down. Current value of {{value}} \n\nThe operation mode of the current nodes can have values of UNKNOWN = 0; STARTING = 1; JOINING = 2; NORMAL = 3; LEAVING = 4; DECOMMISSIONED = 5; DRAINING = 6; DRAINED = 7; MOVING = 8",
```

Comment:
`server` is not in your group by clause of the query. So using {{server.name}} here won't work.